### PR TITLE
Improve/linear layout scene

### PR DESCRIPTION
**Pull request by pleszkiewicz**
# Before -> After

before after

### CODE REVIEW

File: src/test_scenes/sceneLinearLayout.js [hunk 1]
```diff
@@ -1,99 +1,89 @@
...
-  onCenterCenterButtonClick = () => { this.setState({ alignment: 'center-center' }); }
-  onCenterRightButtonClick = () => { this.setState({ alignment: 'center-right' }); }
-  onBottomLeftButtonClick = () => { this.setState({ alignment: 'bottom-left' }); }
-  onBottomCenterButtonClick = () => { this.setState({ alignment: 'bottom-center' }); }
-  onBottomRightButtonClick = () => { this.setState({ alignment: 'bottom-right' }); }
-
-  onWidthToogleChanged = (event) => { this.setState({ width: event.On ? 1 : 0 }); }
-  onHeightToogleChanged = (event) => { this.setState({ height: event.On ? 1 : 0 }); }
+  onOrientationChanged = event => this.setState({ isHorizontal: event.On });
+  onPaddingChanged = (padding) => this.setState({ padding });
```
Comment (kpiascik) [inline src/test_scenes/sceneLinearLayout.js:14]: you don't need to specify `{padding: padding}` and same for `alignment` when you call `setState`?

File: src/test_scenes/sceneLinearLayout.js [hunk 2]
```diff
@@ -1,99 +1,89 @@
...
 
-        <View position={[0, -2.2, 0]}>
+        <LinearLayout
+          orientation={Orientation.vertical}
+          anchorPoint={Alignment.topCenter}
+          defaultItemAlignment={Alignment.topCenter}
+          defaultItemPadding={[0, 0, 0.02, 0]}
+          position={[0.4, -1.3, 0]}
+        >
           <Text position={[0, 0.05, 0]} fontSize={0.08} anchorPoint={'bottom-center'}>Set items alignment:</Text>
```
Comment (kpiascik) [inline src/test_scenes/sceneLinearLayout.js]: you're missing a change to `anchorPoint` here.

File: src/utils/alignment.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Alignment, GridLayout, LinearLayout, Text, Toggle, ToggleGroup, View } from 'magic-script-components';
+import { Alignment, GridLayout, LinearLayout, Orientation, Text, Toggle, ToggleGroup, ToggleType, View } from 'magic-script-components';
```
Comment (kpiascik) [inline src/utils/alignment.js]: we no longer need `View` here.

File: src/utils/padding.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { View, Text, Toggle, LinearLayout,ToggleGroup } from 'magic-script-components';
+import { Alignment, LinearLayout, Orientation, Text, Toggle, ToggleGroup, ToggleType, View } from 'magic-script-components';
```
Comment (kpiascik) [inline src/utils/padding.js]: don't need `View` here anymore

File: src/test_scenes/sceneLinearLayout.js (hunk 1 above)
Comment (pleszkiewicz) [reply src/test_scenes/sceneLinearLayout.js:14]: `{ padding: padding }` is eqivalent with `{ padding }`

https://stackoverflow.com/questions/45819825/create-an-object-with-same-key-and-value-without-duplicating-the-data

File: src/utils/alignment.js (hunk 3 above)
Comment (pleszkiewicz) [reply src/utils/alignment.js]: good catch 👍

**Review by kpiascik — APPROVED** [review]

**Pull request merged**